packages/playground/vite.config.ts

```diff
@@ -10,12 +10,23 @@ const config = definePlaygroundViteConfig({
     "@typespec/openapi",
     "@typespec/versioning",
     "@typespec/openapi3",
+    "@typespec/protobuf",
   ],
   samples: {
-    "API versioning": "samples/versioning.tsp",
-    "Discriminated unions": "samples/unions.tsp",
-    "HTTP service": "samples/http.tsp",
-    "REST framework": "samples/rest.tsp",
+    "API versioning": {
+      fileName: "samples/versioning.tsp",
+      preferredEmitter: "@typespec/openapi3",
```

Comment:
Would it not be better to have this as an array of emitters that are compatible with this sample?

Reply:
In this PR I'm hoping to do the very minimum thing to light up typespec/protobuf in the existing playground so that we can share it. If we want to rework the playground a little bit to have an emitter -> sample flow then we should do that separately IMO and just let this be a dead simple autoselect for now.

Reply:
sure lets merge that and update after, can you just make a PR updating typespec-azure with the new flag too